packages/amplify-graphql-schema-generator/src/__tests__/__snapshots__/mysql-datasource-adapter.test.ts.snap

```diff
@@ -8,7 +26,7 @@ exports[`testDataSourceAdapter test generate graphql schema from internal repren
 }
 
 type Country @model {
-  id: Int! @default(value: \\"uuid()\\") @primaryKey
+  id: Int @primaryKey
```

Comment:
Shouldn't the primary key fields required? Currently MySQL doesn't support `RETURNING` so we can't retrieve the last inserted record. For this reason, amplify should assign a value to id fields if customer didn't pass on the input.

Reply:
I think that could cause problems for some database designs couldn't it? If the customer has set up an auto-incrementing default value as the primary key, they may actually rely on that auto-incrementing design. Maybe that's just a tradeoff here and we won't be able to reasonably solve it with current platform(s) but it's a thought

Reply:
Updated this case to retain the primary key fields as required, even if there is an auto increment default attached to it. This will align with the current behavior of CPK where they are required as input from the client. [Amplify doc link for CPK](https://docs.amplify.aws/cli/graphql/data-modeling/#configure-a-primary-key).